nexus/test-utils/src/lib.rs

```diff
@@ -118,7 +118,7 @@ pub struct ControlPlaneTestContext<N> {
     pub sled_agent2: sim::Server,
     pub oximeter: Oximeter,
     pub producer: ProducerServer,
-    pub gateway: HashMap<SwitchLocation, GatewayTestContext>,
+    pub gateway: BTreeMap<SwitchLocation, GatewayTestContext>,
```

Comment:
Just curious, why switch the map type here?

Reply:
This is what causes `omicron-dev run-all` to print the MGS addresses in a consistent order (switch0, then switch1).